.github/workflows/ci.yml

```diff
@@ -179,19 +180,31 @@ jobs:
       - name: Install scarb
         uses: software-mansion/setup-scarb@v1
         with:
-          scarb-version: "2.6.4"
+          scarb-version: "2.7.0"
```

Comment:
Shouldn't this (and the others `2.7.0`) be `2.7.1`?

Reply:
Done!